deliveries/docker/apache-ignite/s390x/Dockerfile

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Base Image
+FROM  s390x/ubuntu:18.04
+
+ARG IGNITE_VER=2.9.1
+
+# The Author
+LABEL maintainer="LoZ Open Source Ecosystem (https://www.ibm.com/community/z/usergroups/opensource)"
+
+# Set Environment Variables
+ENV WORKDIR /root
+ENV JAVA_HOME=$WORKDIR/jdk-11.0.7+10
+ENV IGNITE_HOME=$WORKDIR/apache-ignite-${IGNITE_VER}-bin
+ENV PATH=$JAVA_HOME/bin:$WORKDIR/apache-ignite-${IGNITE_VER}-bin/bin:$PATH
+
+# Install Dependencies
+RUN apt-get update &&  apt-get install -y \
+        wget \
+        tar \
+        unzip \
+# Download AdoptOpenJDK 8
+ && cd $WORKDIR  \
+ && wget https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.7%2B10_openj9-0.20.0/OpenJDK11U-jdk_s390x_linux_openj9_11.0.7_10_openj9-0.20.0.tar.gz \
```

Comment:
Get, unpack and cleanup should be united in single layer.
And be separated with standalone RUN command